FirebaseVertexAI/Sources/GenerativeModel.swift

```diff
@@ -163,7 +163,7 @@ public final class GenerativeModel {
   /// [zero-shot](https://developers.google.com/machine-learning/glossary/generative#zero-shot-prompting)
   /// or "direct" prompts. For
   /// [few-shot](https://developers.google.com/machine-learning/glossary/generative#few-shot-prompting)
-  /// prompts, see ``generateContent(_:)-58rm0``.
+  /// prompts, see ``generateContent(_:)``.
```

Comment:
nit: `generateContentStream(_ content: @autoclosure () throws -> [ModelContent])` and single backticks.

Reply:
Done.